app/CMakeLists.txt

```diff
@@ -350,8 +350,31 @@ if(CELERITAS_BUILD_DEMOS AND CELERITAS_USE_Geant4)
     demo-geant-integration/SensitiveHit.cc
     demo-geant-integration/TrackingAction.cc
   )
+  set(_demo_geant_integration_libs Celeritas::accel)
+
+  if(CELERITAS_USE_ROOT)
+    list(APPEND _demo_geant_integration_src
+      demo-geant-integration/HitRootIO.cc
+    )
+    list(APPEND _demo_geant_integration_libs ROOT::Tree HitClassesDict)
+    include_directories(
+      ${CMAKE_CURRENT_SOURCE_DIR}/demo-geant-integration
+      ${ROOT_INCLUDE_DIRS} ${Geant4_INCLUDE_DIR}
+    )
+    REFLEX_GENERATE_DICTIONARY(HitClasses
+      demo-geant-integration/HitClasses.hh
+      SELECTION demo-geant-integration/selection.xml
+    )
+    add_library(HitClassesDict SHARED HitClasses.cxx)
+    set(libsuffix .so)
+    set(ROOT_LIBRARY_PROPERTIES ${ROOT_LIBRARY_PROPERTIES} SUFFIX ${libsuffix})
+    set_target_properties(HitClassesDict PROPERTIES ${ROOT_LIBRARY_PROPERTIES})
```

Comment:
Why do you need this? (i.e. how is it used?)

Reply:
Okay, removed unnecessary use of ROOT_LIBRARY_PROPERTIES!